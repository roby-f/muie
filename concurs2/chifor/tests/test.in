10
1 2168116341
1 73754755
1 4942765553
1 804800221
1 810811834
2
1 4741461436
1 4567377025
2
2
