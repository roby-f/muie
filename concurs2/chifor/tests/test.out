2168116341
2168116341
7037127139
7037127139
7043138752
7037127139
10973788354
10973788354
10973788354
7037127139
